swift-sdk/IterableMessaging.swift

```diff
@@ -205,7 +208,8 @@ public extension Notification.Name {
          saveToInbox: Bool = false,
          inboxMetadata: IterableInboxMetadata? = nil,
          customPayload: [AnyHashable: Any]? = nil,
-         read: Bool = false) {
+         read: Bool = false,
+         priorityLevel: Double? = nil) {
```

Comment:
Since we have a default priority level, I think we should set it to `PriorityLevel.default`. The logic will be much easier if we don't have to deal with `nil`.